embeddings/pipeline/flair_hps_pipeline.py

```diff
@@ -104,6 +104,14 @@ def _pop_sampled_parameters(
         assert isinstance(load_model_kwargs, dict)
         return embedding_name, document_embedding, task_train_kwargs, load_model_kwargs
 
+    @staticmethod
+    def _revert_default_hps_task_train_kwargs(
+        task_train_kwargs: Dict[str, ParameterValues]
+    ) -> Dict[str, ParameterValues]:
+        task_train_kwargs["param_selection_mode"] = False
+        task_train_kwargs["save_final_model"] = True
+        return task_train_kwargs
```

Comment:
1. The returned value suggests that the input dict is not being modified.
2. Can the function be modified to `property`?

Reply:
1. Refactored
2. I think not, because we should not modify original dict, as it was before (fixed)

Reply:
2. (opinion) readability